### ISSUE TITLE
Accessibility Improvements Datepicker

### ISSUE THREAD

**Issue by stevenfowler16** [Dogfalo/materialize#6572]
## Expected Behavior
When clicking a date picker a user should be able to easily change the year and month with the keyboard as well as submit.

## Current Behavior

Currently you have to tab through every single day to change the date. It's impossible to select a year or month. 

## Possible Solution

Here is a good example of a date picker that makes using a keyboard easy.
[Date Picker](https://dequeuniversity.com/library/aria/date-picker)
1. You can tab to the main points of the control (Arrows, submit,etc)
2. You can easily switch the year (alt+pageup alt+pagedown)
3. You can easily switch the month(arrows or pageup or pagedown) 
4. You can easily submit the date (enter key)

Here is an WCAG accepted date picker
[WCAG](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html)

## Steps to Reproduce (for bugs)

1. Navigate to https://materializecss.com/pickers.html
2. Open the date picker (have to use the mouse) another issue
3. Try and navigate to a date that is a year in the past without using the mouse

## Context

I have been looking for WCAG accessible date pickers and I actually really like the design of MaterializeCSS but I think it would fail a major WCAG guideline (Level A) specifically 
[2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation.html)

## Your Environment

* Version used: V 1.0.0
* Browser Name and version: Chrome 83.0.4103.116 
* Operating System and version (desktop or mobile): Windows 10
